packages/charts/react-charting/src/components/DeclarativeChart/DeclarativeChart.tsx

```diff
@@ -25,6 +25,13 @@ import { GaugeChart } from '../GaugeChart/index';
 import { GroupedVerticalBarChart } from '../GroupedVerticalBarChart/index';
 import { VerticalBarChart } from '../VerticalBarChart/index';
 
+import { useTheme } from '@fluentui/react';
+
+export const UseIsDarkTheme = (): boolean => {
+  const theme = useTheme();
+  return theme.isInverted;
```

Comment:
is theme guaranteed to be present and not undefined

Reply:
will add a check